src/requests/features.jl

```diff
@@ -304,7 +304,13 @@ function julia_getModuleAt_request(params::TextDocumentPositionParams, server::L
     doc = getdocument(server, URI2(params.textDocument.uri))
     offset = get_offset(doc, params.position)
     x = get_expr(getcst(doc), offset)
-    return x isa EXPR ? get_module_of(StaticLint.retrieve_scope(x)) : "Main"
+    if x isa EXPR
+        scope = StaticLint.retrieve_scope(x)
+        if x !== nothing
```

Comment:
Is this supposed to be `scope !== nothing` ?

Reply:
Whoops, yes, of course.